### PR TITLE
Docs: remove gpu.js dependency

### PR DESCRIPTION
**Pull request by kirszenbaum**
This PR:
- Adjusts the HF docs to #816

### CODE REVIEW

File: docs/guide/enabling-gpu-acceleration.md [hunk 1]
```diff
@@ -1,19 +1,57 @@
...
+:::
 
-In order to take advantage of the GPU acceleration, the GPU.js constructor has to be passed to the engine configuration 
-(supported version of GPU.js is 2.3.0):
+To enable GPU acceleration, follow the steps below.
 
-```javascript
+## Step 1: Use HyperFormula 1.1.0
+
+As `gpu.js` was removed from optional dependencies in [HyperFormula 1.2.0](../guide/release-notes.md#_1-2-0), you can only use GPU acceleration in HyperFormula 1.1.0 (or lower).
```
Comment (jansiegel) [inline docs/guide/enabling-gpu-acceleration.md:11]: @kirszenbaum 
Correct me if I'm wrong, @wojciechczerniak, but I think `gpu.js` support will still be provided in `1.2.0`, but the user will have to install `gpu.js` on their own.

Comment (wojciechczerniak) [reply docs/guide/enabling-gpu-acceleration.md:11]: You're right 👍 For devs nothing changes. Flow is the same, `npm install gpu.js`, `import gpu from 'gpu.js'`, and give it to HF settings `{ gpu: gpu }`. The only change is that NPM didn't respected that last time.

File: docs/guide/dependencies.md [hunk 2]
```diff
@@ -9,7 +9,6 @@ features.
 | [bessel](https://github.com/SheetJS/bessel) | Apache v2.0 | SheetJS |
 | [Chevrotain](https://github.com/SAP/chevrotain) | Apache v2.0 | SAP SE or an SAP affiliate company |
 | [core-js](https://github.com/zloirock/core-js) | The MIT License | Denis Pushkarev |
-| [GPU.js](https://github.com/gpujs/gpu.js/) ([optional](enabling-gpu-acceleration.md)) | The MIT License | GPU.js team |
```
Comment (wojciechczerniak) [inline docs/guide/dependencies.md:12]: Dependency is not yet removed. It's optional until v2.0

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
Comment (wojciechczerniak) [inline CHANGELOG.md:7]: Please keep the `## [Unreleased]` header for future entries

**Review by wojciechczerniak — CHANGES_REQUESTED** [review]

File: docs/guide/enabling-gpu-acceleration.md (hunk 1 above)
Comment (kirszenbaum) [reply docs/guide/enabling-gpu-acceleration.md]: @jansiegel @wojciechczerniak - got it, is this OK then? 5a1de61

File: docs/guide/dependencies.md (hunk 2 above)
Comment (kirszenbaum) [reply docs/guide/dependencies.md]: Done: e8e8323

File: CHANGELOG.md (hunk 3 above)
Comment (kirszenbaum) [reply CHANGELOG.md]: Done: e8e8323

Comment (wojciechczerniak) [reply CHANGELOG.md]: We should have both headers for release

Comment (jansiegel) [reply CHANGELOG.md:7]: @wojciechczerniak Shouldn't it be changed on the release branch, not this PR?

Comment (wojciechczerniak) [reply CHANGELOG.md:7]: Up to you guys, I've seen release notes updated here. Just keep them in sync

Comment (jansiegel) [reply CHANGELOG.md:7]: I'm going to be doing the release branch right after this PR is merged, co it doesn't really matter this time 👍

**Review by wojciechczerniak — APPROVED** [review]
👏

**Pull request merged**